src/commands/assign.js

```diff
@@ -207,7 +235,11 @@ async function checkAssigned() {
         }
         // Add the id of the newly assigned submission to the list of assigned
         // submissions.
-        assigned.push(s.id);
+        assigned.push({
+          submission_id: s.id,
+          project_id: s.project.id,
+          timeToFinish: (12 - moment.utc().diff(s.assigned_at, 'hours')),
```

Comment:
A bit. This: `.filter(s => assigned.indexOf(s.id) === -1)` makes sure we are only pushing assigned reviews that are not already in the `assigned` array. So it only gets pushed to the assigned array once, and therefore the `timeToFinish` property only gets assigned once.